samples/sqltransport/multi-instance/sample.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SqlServer - SQL Transport Mulit-Instance Mode
+summary: SqlServer transport running in Multi-Instance Mode.
+reviewed: 2016-03-25
+tags:
+- SQL Server
+- Multi-Instance Mode
+related:
+- nservicebus/sqlserver/deployment-options
+---
+
+## Prerequisites
+
+ 1. Make sure SQL Server Express is installed and accessible as `.\SQLEXPRESS`. 
+ 1. Create two databases called `receivercatalog` and `sendercatalog`.
+ 1. Make sure that Distributed Transaction Coordinator (DTC) is running. It can be started runnig `net start msdtc` cammand in system console.  
+
+
+## Running the project
+
+ 1. Start the Sender project (right-click on the project, select the `Debug > Start new instance` option).
+ 1. The text `Press <enter> to send a message` should be displayed in the Sender's console window.
+ 1. Start the Receiver project (right-click on the project, select the `Debug > Start new instance` option).
+ 1. Hit `<enter>` in Sender's console window to send a new message.
+ 2. 
+
+## Verifying that the sample works correctly
+
+ 1. The Receiver displays information that an order was submitted.
+ 1. The Sender displays information that the order was accepted.
+
+
+## Code walk-through
+
+This sample contains three projects:
+
+ * EndpointConnectionLookup - A class library containing common code for providing new `SqlConnection` for transport addresses used in the sample.
+ * Sender - A console application responsible for sending the initial `ClientOrder` message and processing the follow-up `ClientOrderAccepted` message.
+ * Receiver - A console application responsible for processing the order message.
+ * Messages - A class library containing message definitions
+
+Sender and Receiver use different catalogs on the same Sql Server instance. Each catalog contains tables representing queues belonging to NServiceBus endpoint.
```

Comment:
I think it'd be good to link "different catalogs" to http://docs.particular.net/nservicebus/sqlserver/deployment-options.

> Sql Server

SQL Server

>  Each catalog contains tables representing queues belonging to NServiceBus endpoint.

The tables representing queues for a particular endpoint are created in the appropriate catalog, i.e. in the `receivercatalog` for the Receiver endpoint and in the `sendercatalog` for the Sender endpoint .